src/serviceregistry/manual/examples.md

```diff
@@ -0,0 +1,2390 @@
+    ],
+    "providerNames": [
+        "temperature-provider1" 
+    ],
+    "serviceDefinitionNames": [
+        "celsius-info"
+    ],
+    "versions": [
+        "1.0.0", "1.0.1"
+    ],
+    "alivesAt": "",
+    "metadataRequirementsList": [
+    ],
+    "interfaceTemplateNames": [
+        "generic-http", "generic-https"
+    ],
+    "interfacePropertyRequirementsList": [
+    ],
+    "policies": [
+    ]
+  }
+  ~~~
+
+  Response body:
+  ~~~
+  {
+    "entries": [
+      {
+        "instanceId": "temperature-provider1::celsius-info::1.0.0",
+        "provider": {
+          "name": "temperature-provider1",
+          "metadata": {
+            "type": "temperature",
+            "scales": [
+              "Kelvin",
+              "Celsius"
+            ],
+            "customizable": false
+          },
+          "version": "2.1.0",
+          "createdAt": "2024-10-24T23:44:09Z",
+          "updatedAt": "2024-10-24T23:44:09Z"
+        },
+        "serviceDefinition": {
+          "name": "celsius-info",
+          "createdAt": "2024-10-24T21:48:36Z",
+          "updatedAt": "2024-10-24T21:48:36Z"
+        },
+        "version": "1.0.0",
+        "expiresAt": "2030-10-11T14:30:00Z",
+        "metadata": {
+          "frequency": 400,
+          "unrestricted-discovery": true
+        },
+        "interfaces": [
+          {
+            "templateName": "generic-https",
+            "protocol": "https",
+            "policy": "NONE",
+            "properties": {
+              "accessAddresses": [
+                "127.0.100.100"
+              ],
+              "basePath": "/celsius-info",
+              "accessPort": 4040
+            }
+          }
+        ],
+        "createdAt": "2024-10-24T23:47:51Z",
+        "updatedAt": "2024-10-25T14:48:04Z"
+      }
+    ],
+    "count": 1
+  }
+  ~~~
+
+- **Revoke service:** DELETE /serviceregistry/service-discovery/revoke/{instanceid}
+  
+  Query parameter: temperature-provider1::celsius-info::1.1.0
+
+  ~~~
+  http://localhost:8443/serviceregistry/service-discovery/revoke/temperature-provider1%3A%3Acelsius-info%3A%3A1.1.0
+  ~~~
+
+  Response code: 200
+
+## **Logs**
+/serviceregistry/mgmt/logs
+
+  Request body:
+  ~~~
+  {
+    "pagination": {
+      "page": 0,
+      "size": 5,
+      "direction": "ASC",
+      "sortField": "entryDate"
+    },
+    "from": "2024-10-23T00:00:00Z",
+    "to": "2024-10-24T00:00:00Z",
+    "severity": "DEBUG",
+    "logger": "eu.arrowhead.serviceregistry.init.ServiceRegistryApplicationInitListener"
+  }
+  ~~~
+
+  Response body:
+  ~~~
+  {
+    "entries": [
+      {
+        "logId": "11e0a3ef-9184-11ef-9618-0a0027000004",
+        "entryDate": "2024-10-23T21:16:29.318Z",
+        "logger": "eu.arrowhead.serviceregistry.init.ServiceRegistryApplicationInitListener",
+        "severity": "INFO",
+        "message": "Core system eu.arrowhead.serviceregistry.ServiceRegistrySystemInfo@2404abe2 revoked 0 service(s).",
+        "exception": ""
```

Comment:
remove this empty fields (because of the new annotation on the DTO Jackson will ignore the null or empty fields)